store/src/java/com/zimbra/cs/imap/ImapDaemon.java

```diff
@@ -148,6 +150,17 @@ private static boolean isZimbraImapEnabled() throws ServiceException {
         return false;
     }
 
+    private static void maybeInitEphemeralBackendExtension() throws ServiceException {
+        String url = Provisioning.getInstance().getConfig().getEphemeralBackendURL();
+        if(url != null) {
+            String[] tokens = url.split(":");
+            String backendName = tokens[0];
```

Comment:
we need to check if token is non null before assigning to backendName, otherwise it may cause a NPE